Gems/ScriptCanvas/Code/Include/ScriptCanvas/Execution/Interpreted/ExecutionStateInterpretedPerActivation.cpp

```diff
@@ -6,67 +6,78 @@
  *
  */
 
-#include "ExecutionStateInterpretedPerActivation.h"
-
 #include <AzCore/Script/ScriptContext.h>
 #include <AzCore/Script/lua/lua.h>
+#include <ScriptCanvas/Execution/Interpreted/ExecutionInterpretedAPI.h>
+#include <ScriptCanvas/Execution/Interpreted/ExecutionStateInterpretedAPI.h>
+#include <ScriptCanvas/Execution/Interpreted/ExecutionStateInterpretedPerActivation.h>
 #include <ScriptCanvas/Grammar/PrimitivesDeclarations.h>
 
-#include "Execution/Interpreted/ExecutionInterpretedAPI.h"
-#include "Execution/RuntimeComponent.h"
-
 namespace ScriptCanvas
 {
-    ExecutionStateInterpretedPerActivation::ExecutionStateInterpretedPerActivation(const ExecutionStateConfig& config)
+    ExecutionStateInterpretedPerActivation::ExecutionStateInterpretedPerActivation(ExecutionStateConfig& config)
         : ExecutionStateInterpreted(config)
+        , m_deactivationRequired(false)
+        , m_luaRegistryIndex(LUA_NOREF)
     {}
 
     ExecutionStateInterpretedPerActivation::~ExecutionStateInterpretedPerActivation()
     {
         if (m_deactivationRequired)
         {
             StopExecution();
-            ReleaseExecutionStateUnchecked();
+            ReleaseInterpretedInstanceUnchecked();
```

Comment:
These are renamed to clarify an earlier change in combination with this one, that the Lua reference doesn't refer to the execution state pointer, but to the Lua table that these graphs create an own for their lifetime of execution.